src/coreclr/vm/excep.cpp

```diff
@@ -6052,7 +6063,7 @@ CreateCOMPlusExceptionObject(Thread *pThread, EXCEPTION_RECORD *pExceptionRecord
 
             GCPROTECT_END();
 
-            UNINSTALL_UNWIND_AND_CONTINUE_HANDLER;
+            UNINSTALL_UNWIND_AND_CONTINUE_HANDLER_NO_PROBE(true);
```

Comment:
`NO_PROBE` looks like left-over from stackoverflow handling. It has nothing to do with what the macro does now.

Could you please rename it to something like `INSTALL_UNWIND_AND_CONTINUE_HANDLER_EX` /           `UNINSTALL_UNWIND_AND_CONTINUE_HANDLER_EX` ?

Reply:
Done, I was wondering what the _NO_PROBE originally meant.